store/conf/attrs/zimbra-attrs.xml

```diff
@@ -8379,7 +8397,6 @@ TODO: delete them permanently from here
   <globalConfigValue>TLSv1</globalConfigValue>
   <globalConfigValue>TLSv1.1</globalConfigValue>
   <globalConfigValue>TLSv1.2</globalConfigValue>
-  <globalConfigValue>SSLv2Hello</globalConfigValue>
```

Comment:
It might be nice to have the `SslConnectionFactory` portion of the dumpoutput generated via
`java -jar ${JETTY_HOME}/start.jar jetty.server.dumpAfterStart=true`
as described in https://www.eclipse.org/jetty/documentation/current/configuring-ssl.html if you get a chance as well.